.github/ISSUE_TEMPLATE/propose-a-talk.md

```diff
@@ -0,0 +1,22 @@
+---
+name: Propose a talk
+about: Propose a talk or workshop for upcoming meetup/event
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+<!-- You can submit any talks which are related to Python Programming -->
```

Comment:
@bhavin192 How about this change?

```diff
- <!-- You can submit any talks which are related to Python Programming -->
+ <!-- You can submit any talks which are, but not limited to, Python Programming -->
```

Reply:
Yes, this totally makes sense, thanks a lot for taking look at that

Reply:
I have updated this section please take a look